build.gradle.kts

```diff
@@ -21,6 +21,10 @@ buildscript {
     }
 }
 
+rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin> {
```

Comment:
Is this for running JS tests?

Reply:
I forget why I added this, but proj builds without it so I'm removing